ch00git/05Collaboration.ipynb.py

```diff
@@ -188,6 +189,7 @@
 # git diff
 
 # %% jupyter={"outputs_hidden": false} language="bash"
+# git add Wales.md
 # git commit -am "Translating from the Welsh"
```

Comment:
As we are fixing these, maybe we could go as well and remove the instances of `git commit -a`, which I don't think is a good practice when learning git.